gdk/sys/Cargo.toml

```diff
@@ -56,6 +56,7 @@ rust-version = "1.70"
 [package.metadata.docs.rs]
 rustdoc-args = ["--cfg", "docsrs"]
 features = []
```

Comment:
this should be removed, actually if you removed the whole section gir will generate the correct thing

Reply:
Let's keep things as is for now.